pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/CollectiveDataEdition/RouteLeavingGuardVenueCollectiveDataEdition/RouteLeavingGuardVenueCollectiveDataEdition.tsx

```diff
@@ -0,0 +1,38 @@
+import RouteLeavingGuard, {
+  IShouldBlockNavigationReturnValue,
+} from 'new_components/RouteLeavingGuard'
+
+import LeavingGuardDialog from 'new_components/LeavingGuardDialog'
+import React from 'react'
+
+export interface RouteLeavingGuardOfferCreationProps {
+  isFormDirty: boolean
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Je me demande si on ne veut pas donner un nom plus générique à la variable
`isormDirty` sous entend qu'il y a forcément un formulaire, et que c'est la seule raison pour laquelle on va "guard"

Peut être `shouldBlock` directement

Reply:
Celui-ci va être utilisé uniquement dans le cas du formulaire d'édition de données EAC du coup pour moi c'est ok :) 
Celui de la créa d'offre utilise d'autres props pour savoir si oui ou non il faut bloquer, en fait la logique se trouve dans les composants qui wrappent `RouteLeavingGuard` :)